lib/moon/design/breadcrumb/collapsed.ex

```diff
@@ -3,12 +3,16 @@ defmodule Moon.Design.Breadcrumb.Collapsed do
 
   use Moon.StatefulComponent
 
-  alias Moon.Icons.ArrowsRight
+  alias Moon.Icon
   alias Moon.Design.Button.IconButton
   alias Phoenix.LiveView.JS
 
   prop(breadcrumbs, :list)
 
+  prop(divider, :string)
```

Comment:
watcha thinking about adding @doc attributes in this module?

Reply:
Will be added later when made headless.